frontend-project/yarn.lock

```diff
@@ -14354,7 +14354,7 @@ prettier@1.18.2:
   resolved "https://registry.yarnpkg.com/prettier/-/prettier-1.18.2.tgz#6823e7c5900017b4bd3acf46fe9ac4b4d7bda9ea"
   integrity sha512-OeHeMc0JhFE9idD4ZdtNibzY0+TPHSpSSb9h8FqtP+YnoZZ1sl8Vc9b1sasjfymH3SonAF4QcA2+mzHPhMvIiw==
 
-prettier@^1.14.3, prettier@^1.15.3, prettier@^1.17.1, prettier@^1.18.2:
+prettier@^1.14.3, prettier@^1.17.1, prettier@^1.18.2:
```

Comment:
Dlaczego?

Reply:
@ad-m yarn to sam zrobił, nie ruszałem nic manualnie, to jest efekt zwykłego yarn install